webscrapbook/scrapbook/common/files.py

```diff
@@ -34,6 +39,10 @@ def __set_file_constants(self):
         self.__constants.META_REGEX     = str(r"^meta([0-9]*)\.js$")
         self.__constants.FULLTEXT_REGEX = str(r"^fulltext([0-9]*)\.js$")
 
```

Comment:
Is there a justification on using __double_leading_underscore class variables and methods? I wonder we can probably use module constants and normal or _single_leading_underscore methods (or module functions) instead.

Reply:
We probably need to parse using regexp, as the prefixing comments and overall wrappings may change across different versions, or even by the user.

Reply:
We usually don't align variables at equal or colon signs.

Reply:
> Is there a justification on using __double_leading_underscore class variables and methods

I just used the class for encapsulation, so I will use  _single_leading_underscore methods and classes instead. I also wanted to use a class instead of just a module because I was thinking maybe in the future this code may be used for many scrapbooks at once. So each instance of the class could be used for a different scrapbook, and if the variables are part of the class then each scrapbook could even be a different version.

> We probably need to parse using regexp

That is definitely true! I will make parsing more robust with a regex.

> We usually don't align variables at equal or colon signs.

I just do that because I think it's more readable, but I can change it.

Reply:
What is done so far: changed to _single_leading_underscore, parse json content with regex.

I'll need you to verify that the regex will be robust enough since the regex will not work if there is any content after the end of the json object containing '})'.